content/docs/releases/v1.1.0.md

```diff
@@ -0,0 +1,84 @@
++++
+title = "Release v1.1.0"
+short = "v1.1.0"
+weight = 1
+date = "2021-11-28"
++++
+
+NSM v1.1.1 is released via a set of [example use cases](https://github.com/networkservicemesh/deployments-k8s/tree/release/v1.1.1).
+
+## Kubernetes Compatibility
+NSM v1.1.1 has been successfully [integration tested](https://github.com/networkservicemesh/integration-k8s-kind/actions/runs/1514862151) with Kubernetes versions:
+
```

Comment:
I think we need to point that it is related to _server_ k8s versions.

Reply:
Could you say more, I don't quit follow.